admin/menues.rb

```diff
@@ -154,7 +156,21 @@ def new
     redirect_to :back, notice: "#{I18n.t('active_admin.menues.form.member_action.notice')} #{@version.event}"
   end
 
-  batch_action :clone, "data-confirm" => I18n.t('active_admin.menues.form.batch_action.confirm_clone') do |selection|
+  member_action :activate_deactivate_menu_item do
+    menu_item = Goldencobra::Menue.find(params[:id])
+    if menu_item.active
```

Comment:
You could potentially test for `nil` before calling `.active` on the object, right?